scripts/get-model-data.py

```diff
@@ -123,6 +123,7 @@
             'supplierId',
             'supplierName',
             'submittedAt',
+            'status',
```

Comment:
This looks good but I haven't seen any of this script csv stuff before. My assumption is that this is using the brief status rather than the brief response status because is defined in line 98 what properties are being retrieved for the brief response.

Assuming my assumption is correct, then this should be good 👍